docs/quickstart/job-queue-a-preemptive.yaml

```diff
@@ -0,0 +1,23 @@
+queue: queue-a
+jobSetId: job-set-1
+jobs:
+  - priority: 1
+    podSpec:
+      priorityClassName: armada-preemptive-low-priority
```

Comment:
The config above doesn't define this priorityClassName.  Is this a default one defined somewhere else?

Reply:
@kannon92 
Does this make more sense - https://github.com/G-Research/armada/pull/1587/files#diff-566e8c3f80ad8aeb86911519a22c2e5d62e7cd341716e98bd21fd92b7cd2361cR1-R11 ?

Reply:
I think we should probably write something in our user guide about this. 

So I'm thinking we add a section to our user guide about preemption.  

1) Use that example for PriorityClass.
2) Reference the example job.

Reply:
I agree , but we don't have a section for features, like User Guide or Manual, that would be a good addon for docs.

ArgoCD has a good example: https://argo-cd.readthedocs.io/en/stable/